src/test/java/org/killbill/billing/plugin/stripe/TestBase.java

```diff
@@ -57,6 +59,38 @@
     protected StripeConfigPropertiesConfigurationHandler stripeConfigPropertiesConfigurationHandler;
     protected StripeDao dao;
 
+    private static Account buildAccount(final Currency currency, final String country) {
+        return buildAccount(currency, UUID.randomUUID().toString(), UUID.randomUUID().toString(), UUID.randomUUID().toString(), UUID.randomUUID().toString(), UUID.randomUUID().toString().substring(0, 16), country);
+    }
+
+    // Currently needed as TestUtils.buildAccount does not create a valid e-mail address
```

Comment:
FYI Filed https://github.com/killbill/killbill-plugin-framework-java/issues/46 for this.